lib/types.js

```diff
@@ -1,12 +1,39 @@
-const isNumber = val => typeof val === 'number';
+const isNumber = value => typeof value === 'number';
+const isString = value => typeof value === 'string';
+const isBoolean = value => typeof value === 'boolean';
+const isArray = value => Array.isArray(value);
+const isObject = value => typeof value === 'object';
+const isFunction = value => typeof value === 'function';
 
-const castToNumber = val => {
-  if(isNumber(val)) return val;
-  const number = Number(val);
-  if(isNaN(number)) throw new CastError(Number, val);
+
+const castToNumber = value => {
+  if(isNumber(value)) return value;
+  const number = Number(value);
+  if(isNaN(number)) throw new CastError(Number, value);
   return number;
 };
```

Comment:
I like that you clarified value instead of val-- it's more readable and easier to maintain as a new person checkin' out the code. Good job!